magda-ckan-connector/aspect-templates/organization-details.js

```diff
@@ -1,6 +1,40 @@
-return {
+let phone = undefined;
+let website = undefined;
+let email = undefined;
+
+if(organization && organization.extras && organization.extras.length){
+    organization.extras.forEach(item=>{
+        switch(item["key"]){
+            case "email" : 
+                email = item["value"];
+                break;
+            case "telephone" : 
+                phone = item["value"];
+                break;
+            case "website" : 
+                website = item["value"];
+                break;
+        }
+    });
+}
+
+const data = {
     name: organization.name,
     title: organization.title,
     description: organization.description,
-    imageUrl: organization.image_display_url || organization.image_url
+    imageUrl: organization.image_display_url || organization.image_url,
+    phone,
+    email,
+    addrStreet : undefined,
+    addrSuburb : undefined,
+    addrState : undefined,
+    addrPostCode : undefined,
+    addrCountry : undefined,
+    website
 };
+
+Object.keys(data).forEach(key=>{
```

Comment:
Why set these keys in the first place if we're gonna delete them?

Reply:
Adding `address` related fields only for making code consistent with all three connectors so someone else laster knows we trying hard to get all fields list here. 
They are probably easy to remove. 
The `Object.keys` loop probably still have to stay --- sometimes, CKAN could return empty string `email`, `phone` & website. 
Remove those keys will make future data availability checking easier.
Please let me know your idea on this.

Reply:
Eh I see, we can't use a typescript type either because we're in javascript-land in the connectors. 😢.

Yeah ok makes sense, it'd be cool if we could make the connectors use typescript at some stage though 🤔